modules/tile-converter/docs/cli-reference/tile-converter.md

```diff
@@ -53,6 +53,51 @@ Run it with the local web server from project directory:
 $ I3sLayerPath="./data/CairoLayer" DEBUG=i3s-server:* npx i3s-server
 ```
 
+## Docker image
+
+Converter also available as docker image in the [visgl/tile-converter](https://hub.docker.com/r/visgl/tile-converter) repo.
+
+Run it to download tile-converter docker image:
+
+```bash
+$ docker pull visgl/tile-converter
+```
+
+To use converter run:
+
+```bash
+$ docker run
+  -v /path/to/output_folder:/loaders-bundle/data \
+  --rm \
+  visgl/tile-converter \
+  --input-type ... \
+  --token ... \
+  --tileset ... \
+  --name ... \
+  --output ... \
+  --max-depth ...
+```
+
+Docker run arguments:
+
+```bash
```

Comment:
Not sure this section should be `bash` formatted

Reply:
Removed bash section